templates/zerver/app/invite_user.html

```diff
@@ -57,6 +57,7 @@ <h3 id="invite-user-label">{% trans %}Invite users to Zulip{% endtrans %}</h3>
                 <button class="button exit small rounded" data-dismiss="modal">{{ _('Cancel') }}</button>
                 <button id="submit-invitation" class="button small rounded sea-green" type="button"
                   data-loading-text="{{ _('Inviting...') }}">{{ _('Invite') }}</button>
+                <input type="hidden" value="Copy-link" class="btn" id="copy_link" data-clipboard-target="#link">
```

Comment:
This ID is also super generic, which is problematic -- we could have or introduce another element with the same ID elsewhere in the codebase, and if so, your JavaScript would behave unexpectedly.